docker/arm/Dockerfile.arm64v8

```diff
@@ -0,0 +1,31 @@
+# Dockerfile to be used in an ARMv7 environment like Raspverry PI 4
+FROM arm64v8/node:slim
+LABEL maintainer="Coderaiser, tea-mo903"
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+COPY package.json /usr/src/app/
+
+RUN apt-get update && apt-get -y install python2.7 python2.7-dev make g++
+
+ENV PYTHON python2.7
```

Comment:
Should it be `python2.7`? Maybe let's just leave `python`? python 2.7 is [almost dead](http://pyfound.blogspot.com/2019/12/python-2-sunset.html?utm_source=feedburner&utm_medium=feed&utm_campaign=Feed%3A+PythonSoftwareFoundationNews+%28Python+Software+Foundation+News%29) :)

Reply:
good point, I can upadate it and test if the dependencies (`gritty -> node-pty -> python`) still work out with Python 3!

Reply:
works with Python 3 too, images are up-to-date